OpenEXR/IlmImf/ImfOpaqueAttribute.h

```diff
@@ -105,9 +105,12 @@ class OpaqueAttribute: public Attribute
     virtual void		copyValueFrom (const Attribute &other);
 
 
+    const int                   dataSize() const { return _dataSize; }
+    const Array<char>&          data() const { return _data; }
+        
   private:
 
-    Array<char>			_typeName;
+    std::string			_typeName;
     long			_dataSize;
     Array<char>			_data;
```

Comment:
This is another funny one, char is a signed value, and has a sizeof(1) and a bitsize of CHAR_BIT. I wonder if now is an opportunity to make _data an Array of std::uint8_t?